ingress/controllers/nginx/nginx/main.go

```diff
@@ -214,6 +214,11 @@ type nginxConfiguration struct {
 	// http://nginx.org/en/docs/http/ngx_http_gzip_module.html
 	UseGzip bool `structs:"use-gzip,omitempty"`
 
+	// Enables or disables the use of the nginx module that provides support for HTTP/2 in secure connections
+	// http://nginx.org/en/docs/http/ngx_http_v2_module.html
```

Comment:
are you actually disabling the module or just not advertising http2 as a supported upgrade protocol? if the latter, please modify the comment.